spec/test_app/test_suite.sh

```diff
@@ -2,5 +2,4 @@
 
 cd $(dirname $0)
 
-find scripts -type f -executable -exec '{}' \;
-
+find scripts -type f -perm -111 -exec bash '{}' \;
```

Comment:
BSD's `find` manpage:
 
     -perm [-|+]mode
             The mode may be either symbolic (see chmod(1)) or an octal number.  If the mode is symbolic, a
             starting value of zero is assumed and the mode sets or clears permissions without regard to the
             process' file mode creation mask.  If the mode is octal, only bits 07777 (S_ISUID | S_ISGID |
             S_ISTXT | S_IRWXU | S_IRWXG | S_IRWXO) of the file's mode bits participate in the comparison.
             If the mode is preceded by a dash (``-''), this primary evaluates to true if at least all of
             the bits in the mode are set in the file's mode bits.  If the mode is preceded by a plus
             (``+''), this primary evaluates to true if any of the bits in the mode are set in the file's
             mode bits.  Otherwise, this primary evaluates to true if the bits in the mode exactly match the
             file's mode bits.  Note, the first character of a symbolic mode may not be a dash (``-'').

It looks like `/mode` is not supported :/. There's `+mode`, which I'm sure does the same thing on GNU find, but that's deprecated on the latter.

I suppose `-mode` is not too bad in this case - we can control the permissions of the test files.